bliss/catalog.py

```diff
@@ -347,12 +358,24 @@ def filter_by_flux(self, min_flux=0, max_flux=torch.inf, band=2):
         flux_mask = (on_nmgy > min_flux) & (on_nmgy < max_flux)
 
         d = {}
+        nan_ok_variables = {
+            "shear",
+            "convergence",
+            "ellipticity",
+            "blendedness",
+            "locs_vsbc",
+            "ellipticity_vsbc",
+            "star_fluxes_vsbc",
+            "galaxy_fluxes_vsbc",
```

Comment:
Better not to have a list of case-study specific variable names here

Reply:
Is it possible to disallow nans from all variables?

Reply:
I've removed the nan test here. nans may be necessary. They are different from 0 and play the role of mask.

I also remove the max_flux in this function. It doesn't function correctly.

Reply:
Sounds good